### PR TITLE
Dockerized and changed dev scripts accordingly 

### PR DESCRIPTION
**Pull request by omritoptix**

### CODE REVIEW

File: scripts/shared.sh [hunk 1]
```diff
@@ -6,25 +6,38 @@ SETTLEMENT_EXECUTABLE=${SETTLEMENT_EXECUTABLE:-dymd}
 SETTLEMENT_CHAIN_ID=${SETTLEMENT_CHAIN_ID:-"local-testnet"}
 SETTLEMENT_RPC=${SETTLEMENT_RPC:-"127.0.0.1:36657"}
 
+# DA CONFIG
+DA_LC_ENDPOINT=${DA_LC_ENDPOINT:-"127.0.0.1:26659"}
+DA_NAMESPACE_ID=${NAMESPACE_ID:-"[0,0,0,0,0,0,255,255]"}
```
Comment (mtsitrin) [inline scripts/shared.sh:11]: @omritoptix  uses NAMESPACE_ID instead DA_NAMESPACE_ID on purpose?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+RUN apt-get update -y && apt-get install -y build-essential
+
+ENV PACKAGES curl make git bash gcc python3 wget
+
+RUN apt-get install -y $PACKAGES
+
+WORKDIR /app
+
+RUN git clone https://github.com/dymensionxyz/dymension && cd dymension && make install
```
Comment (mtsitrin) [inline Dockerfile:15]: why the hub is in the rollapp dockerfile?

File: Dockerfile [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+COPY go.mod go.sum* ./
+
+RUN go mod download
+
+COPY . .
+
+RUN go install ./cmd/rollappd
+
+# Init rollapp script env vars
```
Comment (mtsitrin) [inline Dockerfile:40]: why u need to set those in the dockerfile?

Comment (omritoptix) [reply Dockerfile:40]: it's redundant. Used it before but forgot to remote it. will remove it.

File: scripts/shared.sh (hunk 1 above)
Comment (omritoptix) [reply scripts/shared.sh:11]: good catch! will change.

File: scripts/run_rollapp.sh [hunk 4]
```diff
@@ -1,22 +1,35 @@
+#!/bin/bash
 BASEDIR=$(dirname "$0")
-source "$BASEDIR"/shared.sh
+. "$BASEDIR"/shared.sh
 
+# If the settlement layer is set to dymension, use $SETTLEMENT_CONFIG otherwise use $SETTLEMENT_CONFIG_MOCK
```
[the remark this answers is not in the document]
Comment (omritoptix) [reply scripts/run_rollapp.sh:5]: Created a boilerplate for the settlement config mock as it has a lot of params.
for the da mock there is only one param which is  the blocktime (i.e DA_LAYER_CONFIG=30s) so didn't find it cumbersome enough to add boilerplate code for it.
We can add it at a later stage.

**Review by mtsitrin — APPROVED** [review]

**Pull request merged**